ansible_tower_client.gemspec

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'ansible_tower_client/version'
+require_relative 'lib/ansible_tower_client/version'
```

Comment:
Is `require_relative` always safe? Is there a reason we're not following the
```
$:.push File.expand_path("../lib", __FILE__)
```
Method used in our other gems?

Reply:
Not sure what you mean by "safe" here. It has the advantage of not mangling `$LOAD_PATH`, plus it looks nicer.

Reply:
The `require_relative` method was added at some point in the Ruby 2.x lifecycle, and I think a lot of code predates that, or simply copy-pasta's the older code.

Reply:
@jrafanie Do you know if we prefer one style or `require` over another here?

Reply:
I think `require_relative` is fine.  I don't see why we can't use it here.  

I think the default behavior of adding a gem to your Gemfile is it tries to put the gem's lib directory on the load path.  So, I think the load path change here was redundant.  So, I'm 👍

Reply:
I don't know why we did the load path stuff here though.  😨